lib/src/ui/home_page.dart

```diff
@@ -99,8 +97,8 @@ class HomePageState extends State<HomePage> {
           AsyncSnapshot<Movie> snapshot,
         ) {
           return snapshot.hasData
-              ? buildGrid(
-                  snapshot.data!,
+              ? PopularPage(
```

Comment:
can we rename this widget to PopularMoviesGrid?, it's the builder for the Grid and not a page, so the name should reference to the Grid

Reply:
done